mobile/src/Admin.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2018 DoubleDutch, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { PureComponent } from 'react'
+import { View, StyleSheet, WebView, Text } from 'react-native'
+import { translate as t } from '@doubledutch/rn-client'
+import colors from './colors'
+
+export default class Admin extends PureComponent {
+  render() {
+    const uri = this.updateUrl()
+    return (
+      <View style={{ flex: 1, backgroundColor: '#F5F5F5' }}>
+        <WebView
+          source={{
+            uri,
+          }}
+        />
+        <View style={[s.boxLeft, s.bottom]}>
+          <Text style={s.titleText}>{t('important')}</Text>
+          <Text style={s.desText}>{t('importantInt')}</Text>
+        </View>
+      </View>
+    )
+  }
+
+  updateUrl = () => {
```

Comment:
Why this name for the function?  It sounds like it is doing something, even though it is just returning something without side effects.